rest-api/offline/main.py

```diff
@@ -78,6 +81,20 @@ def import_biobank_samples():
   logging.info('Generated reconciliation report.')
   return json.dumps({'written': written})
 
+@api_util.auth_required(EXPORTER)
+def export_tables():
+  resource = request.get_data()
+  resource_json = json.loads(resource)
+  database = resource_json.get('database')
+  tables = resource_json.get('tables')
+  if not database:
+    raise BadRequest("database is required")
+  if not tables or not type(tables) is list:
```

Comment:
`not type(tables)` always resolves to `true`, because all types are truthy :)

Need `type(tables) is not list`.

Reply:
Oof. Done. :)